src/events.js

```diff
@@ -185,6 +192,20 @@ module.exports = function (ctx) {
   };
 
   function changeMode(modename, nextModeOptions, eventOptions = {}) {
+
+    // While freehand draw mode is active, the cursor should always be shown as a crosshair.
+    if (modename === 'freehand') {
+      ctx.ui.queueMapClasses({ mouse: Constants.cursors.ADD });
+      ctx.ui.updateMapClasses();
+      CM.overrideGetCursorTypeLogic(() => Constants.cursors.ADD);
```

Comment:
is the ctx.ui stuff necessary? I feel like the override should be sufficient.
Also I assume ur using override instead of set because set will yield control of the cursor for the drag event?

Reply:
> is the ctx.ui stuff necessary? I feel like the override should be sufficient.

You're right, the override is sufficient.

> Also I assume ur using override instead of set because set will yield control of the cursor for the drag event?

Yep, exactly